aws/resource_aws_ecs_service_test.go

```diff
@@ -336,17 +331,15 @@ func TestAccAWSEcsService_withRenamedCluster(t *testing.T) {
 				Config: testAccAWSEcsServiceWithRenamedCluster(clusterName, tdName, svcName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAWSEcsServiceExists("aws_ecs_service.ghost", &service),
-					resource.TestMatchResourceAttr(
-						"aws_ecs_service.ghost", "cluster", originalRegexp),
+					testAccCheckResourceAttrRegionalARN("aws_ecs_service.ghost", "cluster", "ecs", "cluster/"+clusterName+"$"),
```

Comment:
This acceptance test update is currently failing due to the trailing dollar signs here and below:

```
--- FAIL: TestAccAWSEcsService_withRenamedCluster (49.10s)
    testing.go:640: Step 0 error: Check failed: Check 2/2 error: aws_ecs_service.ghost: Attribute 'cluster' expected "arn:aws:ecs:us-west-2:*******:cluster/tf-acc-cluster-svc-w-rc-qsjtr902$", got "arn:aws:ecs:us-west-2:*******:cluster/tf-acc-cluster-svc-w-rc-qsjtr902"
```

Rather than assembling an ARN to check the value though, its typically easier to use [`resource.TestCheckResourceAttrPair()`](https://godoc.org/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestCheckResourceAttrPair) to compare state values between two resources, e.g.

```suggestion
					resource.TestCheckResourceAttrPair("aws_ecs_service.ghost", "cluster", "aws_ecs_cluster.default", "arn"),
```